deps/pcre/CMakeLists.txt

```diff
@@ -0,0 +1,137 @@
+INCLUDE(CheckIncludeFile)
```

Comment:
Honestly, I'm not much of a fan of including another dependency into our code which we have to keep up-to-date. I'd rather make this a compile-time option, in the following order:

- libc's `regcomp_l`
- system libpcre
- system libpcre2
- libc's `regcomp`, with the known shortcomings
- built-in `regcomp`, with the known shortcomings